src/libraries/System.ComponentModel.TypeConverter/src/Resources/Strings.resx

```diff
@@ -250,4 +250,7 @@
   <data name="MemberRelationshipService_RelationshipNotSupported" xml:space="preserve">
     <value>Relationships between {0}.{1} and {2}.{3} are not supported.</value>
   </data>
+  <data name="EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization" xml:space="preserve">
+    <value>Enable {0} as an AppContextSwitch to use BinaryFormatter to deserealize a stream</value>
```

Comment:
Low-hanging fruit: misspelled "deserialize"

More importantly, I think it could inadvertently cause harm if we tell people "here's how to aim the gun at your foot" without first telling them "you know that gun is loaded, right?" Ideally this error message wouldn't tell you to set the switch, but it would instead link to an article which provides remediation steps based on the app's scenario. And one of those remediation steps could very well be to set the switch.

Reply:
Yup, I agree. I'm not sure if GH shows it anymore, but way up in some of the first comments, I say that aka.ms/binaryformatter does not show that you can set an AppContextSwitch to fall back to the old behavior, so I just called it out here to make sure that 1) we don't mention it on purpose or 2) we add it to aka.ms/binaryformatter.

Reply:
The error message should just point to the document, without mentioning the switch name. It is desirable for the unsafe switch name to be burried deep in that document so that people are forced to see the warnings before they find the workaround.

Reply:
We should add it to the docs.